dockerfile_scripts/scrape_libs.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
 
+WHOAMI=$(whoami)
 # Only scrape host libs if AWS/NCCL/OFI was built for this image
 if [ -d /container/aws/lib ]
 then
    # See if we mounted in host libs in the expected location
-   host_dir="/host"
+   host_dir="/det_host"
    if [ -d "$host_dir" ]; then
```

Comment:
Looking at lines 12, 13 and 37 it appears that we're adding "/det_libfabric/...." after "/det_host/...." in the $LD_LIBRARY_PATH search order.  I think we'd want to reverse that, so that if the user provides /det_libfrabic to the container, we definitely want to use the libfabric found there, before any possible different versions that might be in /det_host/usr/lib64 or /det_host/usr/local/lib64.  But this may just be a nit-pick.

Reply:
Let's change this in another PR.

Reply:
Rereading this. The /det_libfabric libs are prepended and /det_host /host libs are appended. I think that's what you are seeing no?